build/common.project.props

```diff
@@ -274,4 +276,5 @@
    <ApexProjects Include="$(RepositoryRootDirectory)test\NuGet.Tests.Apex\*\*.csproj" />
   </ItemGroup>
 
-</Project>
+  <Import Project="OptProfV2.props"/>
```

Comment:
nit: Why import this at the bottom? Does it rely on some props we are setting?

Reply:
We rely on `IsXPlat` being defined near the top of the file (on line 4).